homeassistant/components/husqvarna_automower_ble/config_flow.py

```diff
@@ -171,7 +173,20 @@ async def check_mower(
 
         title = await self.probe_mower(device)
         if title is None:
-            return self.async_abort(reason="cannot_connect")
+            if self.source == SOURCE_BLUETOOTH:
+                return self.async_show_form(
+                    step_id="bluetooth_confirm",
+                    data_schema=BLUETOOTH_SCHEMA,
+                    description_placeholders={"name": self.address},
+                    errors={"base": "cannot_connect"},
+                )
+            return self.async_show_form(
+                step_id="user",
+                data_schema=self.add_suggested_values_to_schema(
+                    USER_SCHEMA, {CONF_ADDRESS: self.address}
+                ),
+                errors={"base": "cannot_connect"},
+            )
```

Comment:
This is not aligned with the PR title. I think it's a wanted change, but please update the PR title too. Maybe just "Improve husqvarna_automower_ble config flow", then explain the improvements in the PR description.